src/CognitiveServices/CognitiveServices.Test/ScenarioTests/CognitiveServicesAccountTests.cs

```diff
@@ -42,13 +42,6 @@ public void TestNewAccount()
             TestController.NewInstance.RunPsTest(traceInterceptor, "Test-NewAzureRmCognitiveServicesAccount");
         }
 
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
-        public void TestNewAccountWithInvalidName()
```

Comment:
Why you deleted these test cases?

Reply:
`NewAzureRmAllKindsOfCognitiveServicesAccounts` is removed because its been outdated for a long time. Back to old time we had stabled Kinds and was defined as enum so we have this test. But now we have more than 30 kinds and keep adding and removing, it doesn't make sense to have this test anymore.

`TestNewAccountWithInvalidName` is removed because I think this is out of scope of PowerShell. The validation happens in RP and it isn't PowerShell's duty to test.